app/src/main/java/io/github/droidkaigi/confsched2018/presentation/sessions/feedback/SessionsFeedbackViewModel.kt

```diff
@@ -51,14 +57,33 @@ class SessionsFeedbackViewModel @Inject constructor(
                 .addTo(compositeDisposable)
     }
 
-    fun onSubmit(sessionFeedback: SessionFeedback) {
-        (session.value as? Result.Success)?.data?.also {
-            repository.submitSessionFeedback(it, sessionFeedback)
-                    .subscribeBy(onError = defaultErrorHandler())
-                    .addTo(compositeDisposable)
+    fun onSubmitClick() {
+        val sessionFeedback = (sessionFeedback.value as? Result.Success)!!.data
```

Comment:
This is my hobby, but I think the direction of data is easier to understand in one direction.

```
viewmode.livedata -> view -> viewmodel.method
```

So, I want to pass SessionFeedback by method parameter  🙇

Reply:
I will return the argument 👍